frontend/.eslintrc.js

```diff
@@ -194,6 +194,7 @@ module.exports = {
         "newlines-between": "always",
       },
     ],
+    "streamlit-custom/no-null-equality": "error",
```

Comment:
any suggestions for a better name here?
- no-null-double-equality
- use-strict-null-equality-checks
- ensure-use-typeguards

Reply:
I like `use-strict-null-equality-checks`

Reply:
yep, `use-strict-null-equality-checks` sounds great!